app/layout.tsx

```diff
@@ -29,15 +28,14 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <script async src="https://visits.bitcoindevs.xyz/script.js" data-website-id="4ee856e4-582c-4a9c-a1c5-10757123803e"></script>
+      <script
+        async
+        src="https://visits.bitcoindevs.xyz/script.js"
+        data-website-id="4ee856e4-582c-4a9c-a1c5-10757123803e"
+      />
```

Comment:
Removing the closing script tag is being done on purpose here?

Reply:
Yeah, since it's self closing, we don't need that other tag